webapp/components/Registration/RegistrationItemCreateUserAccount.vue

```diff
@@ -0,0 +1,321 @@
+<template>
```

Comment:
I really would love to see this page split into several pages as suggested by the design. What do you think?

-> Furthmore I believe that those componenents should be named something like `RegisterationSlide` or `RegistrationPage` not `RegistrationItem`

Reply:
I have not to much heart in that it must be this big. And the base card could have a bit more width to give the user a better overview.
This way it is split into kind of logical steps …

I would like to rename the components as well. 😍👍🏼